### ISSUE TITLE
As a user, I want to be able to log out.

### ISSUE THREAD

**Issue by andyfoster** [Railsn00bs/the-wall#25]
Enough said. Is there something in devise for this?

**Comment by Widdershin** [comment]
Yeah, check out [this guide](https://github.com/plataformatec/devise/wiki/How-To:-Add-sign_in,-sign_out,-and-sign_up-links-to-your-layout-template) :)
